docs/api/shell.md

```diff
@@ -28,6 +28,14 @@ Show the given file in a file manager. If possible, select the file.
 
 * `fullPath` String
 
+Returns `Promise<String>` - Resolves with description corresponding to the error if the operation failed, or empty string if the operation succeeded.
+
+Open the given file in the desktop's default manner.
+
+### `shell.openItemSync(fullPath)`
```

Comment:
The document should explain what `Sync` means here. It does not mean the API would always wait until the opening application exits, it means the API would only wait on Linux for certain applications.

We probably do not need this API at all, as the behavior of this API is confusing, and no one has requested for it.